aws/modules/infrastructure_modules/stacks_app/app.tf

```diff
@@ -0,0 +1,28 @@
+# Create DNS Zones
+module "zones" {
+
+  source  = "terraform-aws-modules/route53/aws//modules/zones"
+  version = "~> 2.0"
+  count   = var.enable_zone
+  zones   = var.public_zones
+  tags    = var.tags
+}
+
+
+# Create DynamoDB
+module "dynamodb_table" {
```

Comment:
Do we want to specify a version for this module?

Reply:
Updated now